javascript-data-model/order-history.js

```diff
@@ -0,0 +1,30 @@
+var orderHistory = [
+  {
+    orderNumber: '001',
+    orderPlaced: '03 / 16 / 2021',
+    booksOrdered: [{
```

Comment:
nice idea using a separate property for the different type of items! so the next step in that idea is to create a property named `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

you will notice you already hit this idea, but this update will allow you have to any number of different item types without having to update the model between orders.